packages/test-utils/src/testUtils/msw/registerMswTestHooks.ts

```diff
@@ -20,11 +20,11 @@
  * @param worker - service worker
  */
 export function registerMswTestHooks(worker: {
-  listen: (t: any) => void;
+  listen: (options?: any) => void;
   close: () => void;
   resetHandlers: () => void;
 }) {
-  beforeAll(() => worker.listen({ onUnhandledRequest: 'error' }));
```

Comment:
Hmm, is it possible to keep this behavior intact somehow? Wouldn't want to break existing tests that rely on this.

Thinking worst case we can perhaps detect whether it's a v1 or v2 worker and pass the option only when needed?

Btw, what's the new behavior for this in v2?

Reply:
Reverted now.